### PR TITLE
i18n(es): Updated `image.mdx` & `netlify.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `image.mdx` & `netlify.mdx` based on this commit 8fbd915f1a6ac3da4d827490655fccc431ce22f8

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/image.mdx [hunk 1]
```diff
@@ -10,9 +10,9 @@ i18nReady: true
 
 import Video from '~/components/Video.astro';
 
-> ⚠️ ¡Esta integración es todavía experimental! Actualmente sólo es compatible con entornos de nodos, ¡pero estate atento a la compatibilidad con Deno en el futuro!
+> ⚠️ ¡Esta integración será archivada en Astro v3.0 (Otoño 2023) en favor del módulo `astro:assets`. Por favor, consulta la [documentación de Assets](/es/guides/assets/) para más información!
 
-Esta **[integración de Astro][astro-integration]** facilita la optimización de imágenes en tu [proyecto de Astro](https://astro.build/), ¡con soporte completo para construcciones de SSG (Static Site Generator) y renderizado en el lado del servidor!
+Esta **[integración de Astro][astro-integration]** optimiza imágenes en tu [proyecto de Astro](https://astro.build/). Es compatible con Astro v2 solo para todos los sitios estáticos y para [algunos hosts de implementación de renderizado en el lado del servidor](#instalacion).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/image.mdx:15]:
```suggestion
Esta **[integración de Astro][astro-integration]** optimiza las imágenes en tu [proyecto de Astro](https://astro.build/). Es compatible con Astro v2 solo para todos los sitios estáticos y para [algunos hosts de implementación de renderizado en el lado del servidor](#instalacion).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

File: src/content/docs/es/guides/integrations-guide/image.mdx [hunk 2]
```diff
@@ -10,9 +10,9 @@ i18nReady: true
 
 import Video from '~/components/Video.astro';
 
-> ⚠️ ¡Esta integración es todavía experimental! Actualmente sólo es compatible con entornos de nodos, ¡pero estate atento a la compatibilidad con Deno en el futuro!
+> ⚠️ ¡Esta integración será archivada en Astro v3.0 (Otoño 2023) en favor del módulo `astro:assets`. Por favor, consulta la [documentación de Assets](/es/guides/assets/) para más información!
 
-Esta **[integración de Astro][astro-integration]** facilita la optimización de imágenes en tu [proyecto de Astro](https://astro.build/), ¡con soporte completo para construcciones de SSG (Static Site Generator) y renderizado en el lado del servidor!
+Esta **[integración de Astro][astro-integration]** optimiza las imágenes en tu [proyecto de Astro](https://astro.build/). Es compatible con Astro v2 solo para todos los sitios estáticos y para [algunos hosts de implementación de renderizado en el lado del servidor](#instalacion).
```
Comment (Yan-Thomas) [inline src/content/docs/es/guides/integrations-guide/image.mdx:15]:
```suggestion
Esta **[integración de Astro][astro-integration]** optimiza las imágenes en tu [proyecto de Astro](https://astro.build/). Es compatible con Astro v2 solo para todos los sitios estáticos y para [algunos hosts de implementación de renderizado en el lado del servidor](#instalación).
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**